bin/workflows/pycbc_make_offline_search_workflow

```diff
@@ -523,9 +533,10 @@ for key in final_bg_files:
         closed_page = [(snrifar_cb, ifar_cb)]
     else:
         closed_page = [(snrifar_cb,)]
-    table = wf.make_foreground_table(workflow, bg_file, hdfbank, open_dir,
-                                     singles=insps, extension='.html',
-                                     tags=bg_file.tags)
+    if len(ifo_sets) > 1:
```

Comment:
what exactly is the foreground table? something in the report? are we sure its something we only want when running with 3 (or more) ifos?